superset-frontend/src/components/MetadataBar/dataset/useDatasetMetadataBar.tsx

```diff
@@ -0,0 +1,103 @@
+/**
```

Comment:
Can you move this feature to `src/datasets/useDatasetMetadataBar`? Check [this](https://github.com/apache/superset/discussions/22330#discussioncomment-4410665) discussion for context.

Reply:
Done. And it looks like it's the first file in this new structure :) `src/features/datasets/hooks/useDatasetMetadataBar`